test/framework/src/main/java/org/elasticsearch/test/ESIntegTestCase.java

```diff
@@ -1519,6 +1524,12 @@ private void postIndexAsyncActions(String[] indices, List<CountDownLatch> inFlig
         boolean supportsDedicatedMasters() default true;
 
         /**
+         * Normally the cluster will auto manage the {@link ElectMasterService#DISCOVERY_ZEN_MINIMUM_MASTER_NODES_SETTING} automatically
```

Comment:
I would slightly rephrase to:
The cluster automatically manages the {@link ElectMasterService#DISCOVERY_ZEN_MINIMUM_MASTER_NODES_SETTING} by default as nodes are started and stopped. Set this to false to manage the setting manually.